.gitignore

```diff
@@ -1,8 +1,12 @@
 .h5
+*.h5
```

Comment:
Are these necessary, since `.h5` is already specified?

Reply:
As far as I understood, the `.h5` rule will not ignore `model.h5` for example, which is why I had added `*.h5`. However, I heard from @KristinaUlicna that she'd like to include an example trained model for the user to play with, so I can change this anyway.